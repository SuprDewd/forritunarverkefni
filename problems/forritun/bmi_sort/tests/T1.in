2

Alice
1.85
75

Bob
1.96
94
