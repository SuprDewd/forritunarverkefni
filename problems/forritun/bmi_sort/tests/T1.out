Alice 21.9
Bob 24.5
